src/Xamarin.Android.Build.Tasks/Utilities/MavenExtensions.cs

```diff
@@ -0,0 +1,229 @@
+using System;
+using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Xml;
+using System.Xml.Serialization;
+using MavenNet;
+using MavenNet.Models;
+using Microsoft.Android.Build.Tasks;
+using Microsoft.Build.Utilities;
+
+namespace Xamarin.Android.Tasks;
+
+static class MavenExtensions
+{
+	static readonly char [] separator = [':'];
+
+	/// <summary>
+	/// Shortcut for !string.IsNullOrWhiteSpace (s)
+	/// </summary>
+	public static bool HasValue ([NotNullWhen (true)] this string? s) => !string.IsNullOrWhiteSpace (s);
+
+	// Helps to 'foreach' into a possibly null array
+	public static T [] OrEmpty<T> (this T []? value)
+	{
+		return value ?? Enumerable.Empty<T> ().ToArray ();
```

Comment:
There should be an `Array.Empty<T>()`:
```suggestion
		return value ?? Array.Empty<T> ();
```